webapp/graphite/storage.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, finders=None, hosts=None):
 
     if hosts is None:
       hosts = settings.CLUSTER_SERVERS
-    remote_hosts = [host for host in hosts if not is_local_interface(host)]
+    remote_hosts = [host for host in hosts if not settings.REMOTE_EXCLUDE_LOCAL or not is_local_interface(host)]
```

Comment:
@bmhatfield would you prefer a `if not (... or ...)` or do you just want an explanation of the logic?

Basically it's just including hosts if either is_local_interface() is not enabled by config (REMOTE_EXCLUDE_LOCAL) or is_local_interface() returns false.